OperatingSystem-Services/Service-SOC/BasicDocumentation/Wazuh-Docker-Single-Node.md

```diff
@@ -0,0 +1,20 @@
+# How to Deploy Wazuh on Docker, Single-Node
+
+
+
+## Get wazuh-docker github repo
+Make sure this is latest version !!
+
+`git clone https://github.com/wazuh/wazuh-docker.git -b v4.6.0`
+
+## Get Certs
+Navigate to the single node directory
+`docker compose -f generate-indexer-certs.yml run --rm generator`
+
+## Deploy docker containers
+
+`docker compose up -d`
```

Comment:
Where do we run this. What does it do? I see the header. What Containers does it create.